src/main/java/net/snowflake/ingest/streaming/internal/StreamingIngestResponseCode.java

```diff
@@ -0,0 +1,114 @@
+  ERR_MISSING_CHANNEL_NAME_IN_REQUEST(18, "A channel name was not supplied in the request"),
+  ERR_CHANNEL_NO_LONGER_EXISTS(19, "The requested channel no longer exists"),
+  ERR_CHANNEL_HAS_INVALID_CLIENT_SEQUENCER(
+      20, "An invalid client sequencer was provided, please reopen the channel"),
+  ERR_CHANNEL_HAS_INVALID_ROW_SEQUENCER(
+      21, "The client provided an out-of-order message, please reopen the channel"),
+  ERR_OTHER_CHANNEL_IN_CHUNK_HAS_ISSUE(
+      22,
+      "A channel whose data was written to in the same blob has an issue and this channel cannot"
+          + " persist data"),
+  ERR_DUPLICATE_CHANNEL_IN_CHUNK(
+      23, "A malformed request was sent that had the same channel within a chunk"),
+  ERR_MISSING_CLIENT_SEQUENCER_IN_REQUEST(
+      24, "A client sequencer is needed in the request payload"),
+  ERR_CHANNEL_DOES_NOT_EXIST_OR_IS_NOT_AUTHORIZED(
+      25, "The channel does not exist or is not authorized"),
+  ROW_SEQUENCER_IS_COMMITTED(26, "The requested row sequencer is committed"),
+  ROW_SEQUENCER_IS_NOT_COMMITTED(27, "The requested row sequencer is not committed"),
+
+  ERR_MISSING_CHUNK_MD5_IN_REQUEST(28, "The request is missing an MD5 for a chunk"),
+  ERR_MISSING_EP_INFO_IN_REQUEST(29, "The request is missing EP information for a chunk"),
+  ERR_INVALID_CHUNK_LENGTH(30, "An invalid chunk length was provided"),
+  ERR_INVALID_ROW_COUNT_IN_EP_INFO(31, "The EP information contains an invalid row count"),
+  ERR_INVALID_COLUMN_IN_EP_INFO(32, "The EP information contains invalid column information"),
+  ERR_INVALID_EP_INFO_GENERIC(33, "The EP information is malformed"),
+  ERR_INVALID_BLOB_NAME(34, "The Client SDK generated an invalid blob key"),
+
+  ERR_CHANNEL_MUST_BE_REOPENED(36, "The channel must be reopened"),
+  ERR_MISSING_ROLE_IN_REQUEST(37, "The role name is missing in the request"),
+  ERR_BLOB_HAS_WRONG_FORMAT_OR_EXTENSION(
+      38, "The blob being registered has the wrong key format or extension"),
+  ERR_BLOB_MISSING_MD5(39, "The blob is missing a MD5"),
+  ERR_MISSING_COLUMN_IN_EP_INFO(
+      40,
+      "A column in the table was not provided in the EP information. A schema change most likely"
+          + " occurred on the table, please re-open the channel"),
+  ERR_DUPLICATE_BLOB_IN_REQUEST(41, "Duplicated blob in request"),
+  ERR_UNSUPPORTED_BLOB_FILE_VERSION(42, "Unsupported blob file version"),
+  ERR_OUTDATED_CLIENT_SDK_VERSION(
+      43, "Outdated Client SDK. Please update your SDK to the latest version"),
+  ERR_UNDEFINED_USER_AGENT_HEADER(44, "Unspecified User-Agent request header"),
+  ERR_MALFORMED_USER_AGENT_HEADER(45, "Malformed User-Agent request header"),
+  ERR_INTERLEAVING_NOT_SUPPORTED_AT_THIS_TIME(
+      46, "Interleaving among tables is not supported at this time"),
+  ERR_RESOLVED_TABLE_IS_READ_ONLY(47, "Table is read-only"),
+  ERR_INVALID_VAL_IN_EP_INFO(48, "The request contains invalid values in the EP information"),
```

Comment:
Hmmmm maybe say, `The request contains invalid column metadata, please contact Snowflake support`.